bvm/ballerina-core/src/main/java/org/ballerinalang/model/values/BValueType.java

```diff
@@ -18,6 +18,8 @@
 
 package org.ballerinalang.model.values;
 
+import org.ballerinalang.model.types.BType;
+
```

Comment:
Shall we remove this extra line added after the import?

Reply:
Done.